config/awaymissionconfig.txt

```diff
@@ -17,4 +17,6 @@ _maps/RandomZLevels/away_mission/jungleresort.dmm
 #_maps/RandomZLevels/away_mission/caves.dmm
 #_maps/RandomZLevels/away_mission/snowdin.dmm
 #_maps/RandomZLevels/away_mission/research.dmm
-#_maps/RandomZLevels/away_mission/SnowCabin.dmm
\ No newline at end of file
+#_maps/RandomZLevels/away_mission/SnowCabin.dmm
+modular_splurt/_maps/away_mission/AGRComplexClock.dmm
+modular_splurt/_maps/away_mission/AGRComplexCult.dmm
```

Comment:
Please comment these two to keep the repo configuration file without maps by default

Reply:
Done! Left jungleresort though, since that one was like that prior.